localstack/aws/api/s3/__init__.py

```diff
@@ -932,6 +932,15 @@ class InvalidLocationConstraint(ServiceException):
     LocationConstraint: Optional[BucketRegion]
 
 
+class EntityTooLarge(ServiceException):
```

Comment:
I looked at the `EntityTooSmall` class. It does have different keys, but the snapshot appears the be identical. Is it just some more of the S3 patch weirdness or an outdated scaffold?

Reply:
Nice check! This is because I've already added `EntityTooSmall` earlier as this exception is also raised by `CompleteMultipartUpload` when a part which is not the last part is under 5Mb... so I could add the specific one here, but reused it. The shape is similar. Good catch 😄